src/Microsoft.AspNetCore.Mvc.Core/Infrastructure/NullableCompatibilitySwitch.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Mvc.Infrastructure
+{
+    internal class NullableCompatibilitySwitch<TValue> : ICompatibilitySwitch where TValue : struct
```

Comment:
Why does this class still exist? Put another way, what's wrong with `CompatibilitySwitch<int?>`?

Reply:
`CompatibilitySwitch` does not allow a nullable value.

Reply:
Sigh :ok: